src/support/base64.h

```diff
@@ -293,6 +293,40 @@ class Base64OutStream : public dmlc::Stream {
     }
   }
 };
+
+inline size_t b64strlen(const std::string b64str) {
+  ICHECK(b64str.size() % 4 == 0) << "invalid base64 encoding";
```

Comment:
Move these two functions to callers, since they are not necessarily the optimal way of decoding b64. The b64Stream is the preferred way. So avoid making them in support(and gradually we move the contrib impl over, or keep them self-contained to a sub module scope) is better

Reply:
Ah, I see. I moved them to contrib side (`src/contrib/torch/tvm_module_wrapper/RuntimeModuleWrapperTVM.cc`) since it is the only user currently.